### ISSUE TITLE
using with google slides

### ISSUE THREAD

**Issue by kameshsampath** [excalidraw/virgil#70]
is there a way to use this font with Google Slides ?

**Comment by boston2029** [comment]
I don't believe so. You can't use custom fonts on Google Slides, I believe. The only way you could do this is by testing it out in a place you can, and then screenshotting it and putting it in the slide.
